app/controllers/anonymous_feedback_controller.rb

```diff
@@ -23,4 +31,20 @@ def index
   def set_requester_on(request)
     # this is anonymous feedback, so requester doesn't need to be set
   end
+
+private
+  def index_params
+    params.permit(:path, :page)
+  end
+
+  def fetch_anonymous_feedback_from_support_api
+    api_params = { path_prefix: index_params[:path] }.tap {
```

Comment:
The `tap` part of this requires a bit too much reading to understand, imo:
```ruby
api_params = { path_prefix: index_params[:path] }
api_params[:page] = index_params[:page] if index_params[:page]
support_api.anonymous_feedback(api_params)
```
looks cleaner imo.